elasticsearch/sgconfig/sg_rolesmapping.yml

```diff
@@ -0,0 +1,15 @@
+sg_role_kibana:
+  users:
+    - 'CN=system.logging.kibana,OU=OpenShift,O=Logging,L=Test,C=DE'
```

Comment:
@richm  should we just leave it at 'CN=system.logging.kibana,OU=OpenShift,O=Logging' ? Any preferences on L and C if we should have them...